### PR TITLE
[DOCU-2771] Konnect OIDC config

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Summary

OIDC is now supported in Konnect. We need to update our SSO documentation for Okta and create a new document for Azure (work currently blocked) and generic OIDC. 
### Reason

[DOCU-2771](https://konghq.atlassian.net/browse/DOCU-2271)
### Testing

Questions for reviewers:
- Does anything feel missing from the generic OIDC doc?
- Does the title work for the generic OIDC doc?
- In the Okta doc, I kind of feel like the "Log in through Okta to test the integration" section is mostly a repeat of the "Test and apply the configuration" section. I'm thinking they could be combined but wanted a second opinion. 
- In the Okta doc, I was thinking that it would be nice (not 100% needed) to break the "Prerequisites and overview of steps" section into "Prerequisites" and "Overview of steps". The overview is nice, but since the doc covers how to complete all those steps, I don't think they're a prerequisite. Thoughts?

### CODE REVIEW

File: app/konnect/org-management/oidc-idp.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+authentication.
+
+## Prerequisites
+
+* {{site.konnect_short_name}} must be added to your IdP
+* Claims are set up in your IdP
+
+## Set up SSO in {{site.konnect_short_name}}
+
+1. In [{{site.konnect_saas}}](https://cloud.konghq.com), click ![](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
```
Comment (Guaris) [inline app/konnect/org-management/oidc-idp.md:26]:
```suggestion
1. In [{{site.konnect_saas}}](https://cloud.konghq.com), click {% konnect_icon cogwheel %}
```

File: app/konnect/org-management/oidc-idp.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    also signs users out of their IdP session.
+
+1. Click **Save**.
+
+## Test and apply the configuration
+
+{:.important}
+> **Important:** Keep built-in authentication enabled while you are testing IdP authentication. Only disable built-in authentication after successfully testing IdP authentication.
+
+You can test the SSO configuration by navigating to the login URI based on the Organization Login Path you set earlier. For example: `cloud.konghq.com/login/examplepath`. You will see the IdP sign in window if your configuration is set up correctly.
```
Comment (Guaris) [inline app/konnect/org-management/oidc-idp.md:61]:
```suggestion
You can test the SSO configuration by navigating to the login URI based on the organization login path you set earlier. For example: `cloud.konghq.com/login/examplepath`. If your configuration is set up correctly, you will see the IdP sign-in window.
```

**Review by Guaris — APPROVED** [review]

**Comment by rosskukulinski** [comment]
Hi @Guaris - I'm curious why this is part of the Gateway 3.0 milestone?

**Comment by Guaris** [comment]
@rosskukulinski  A mistake. This PR is actually just waiting on Hayden's approval.

**Pull request merged**